qa/tasks/cephfs/kernel_mount.py

```diff
@@ -26,9 +26,11 @@ def __init__(self, ctx, test_dir, client_id, client_remote,
             client_keyring_path=client_keyring_path, hostfs_mntpt=hostfs_mntpt,
             cephfs_name=cephfs_name, cephfs_mntpt=cephfs_mntpt, brxnet=brxnet)
 
+        self.dynamic_debug = config.get('dynamic_debug', False)
         self.rbytes = config.get('rbytes', False)
         self.inst = None
         self.addr = None
+        self.ctx = ctx
```

Comment:
BTW: CephFSMount already stores the `ctx` for you.

Reply:
Cool, I will remove it here.